src/adapter/src/coord/message_handler.rs

```diff
@@ -240,15 +239,11 @@ impl<S: Append + 'static> Coordinator<S> {
                     // no better solution presents itself. Possibly sinks should
                     // have an error bit, and an error here would set the error
                     // bit on the sink.
-                    self.handle_sink_connection_ready(
-                        id,
-                        oid,
-                        connection,
-                        compute_instance,
-                        Some(&session),
-                    )
-                    .await
-                    .expect("sinks should be validated by sequence_create_sink");
+                    self.handle_sink_connection_ready(id, oid, connection, Some(&session))
+                        .await
+                        // XXX(chae): I really don't like this -- especially as we're now doing cross
```

Comment:
this comment is not true anymore, right?

Reply:
It is true! `handle_sink_connection_ready` is what has the call to create the storaged instance!